tests/nested/manual/split-refresh/task.yaml

```diff
@@ -63,9 +63,6 @@ execute: |
   # For now pretend we have a modeenv
   echo "mode=run" > /var/lib/snapd/modeenv
   tests.cleanup defer rm -f /var/lib/snapd/modeenv
```

Comment:
You do not need to create modeenv either. Also, mind using this PR to remove this thing from the muinstaller test?

Reply:
Sure, I'll remove that in a separate commit and force-push to clean up the history (1 commit for the new test and another for the muinstaller-real test)